src/db/index.ts

```diff
@@ -0,0 +1,39 @@
+import { DataTypes } from 'sequelize';
+import UsersAttributes from '../types/usersAttributes';
+import sequelize from './connection';
+
+const Users = sequelize.define<UsersAttributes>('users', {
+  id: {
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  fullName: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  password: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: false,
```

Comment:
the default is false, SO no need to specify the value
```suggestion
```

Reply:
done.